package.json

```diff
@@ -45,7 +45,7 @@
     "build": "grunt",
     "module": "git submodule init && git submodule update",
     "start": "source env/bin/activate && python run.py",
-    "virtualenv" : "pip install virtualenv && virtualenv env &&  source env/bin/activate",
+    "virtualenv" : "pip install virtualenv && virtualenv env &&  npm run pip && source env/bin/activate",
```

Comment:
nit: two spaces between `&&` and `npm`.

We want to install the packages into the virtual environment, so this command should come after `source env/bin/activate`.

As is though, this will mean the `npm run init` command will double install pip stuff, once before the virtualenv is active, and once after:

` "init": "npm run module && npm run virtualenv && npm run pip && npm install && npm run config",`